scripts/entrypoint.pl

```diff
@@ -174,8 +315,6 @@ sub write_localconfig {
         }
     }
 
-    unlink($filename);
```

Comment:
This change caught my eye, so I wanted to check in and make sure it's supposed to be here, but otherwise I'm not equipped to say whether this is a good patch or not — too much here is changed that I simply have no context for and can't test readily.